environments/infrastructure/configuration.yml

```diff
@@ -12,20 +12,21 @@ cephclient_mons:
 
 homer_host: "{{ hostvars[inventory_hostname]['ansible_' + hostvars[inventory_hostname]['internal_interface']]['ipv4']['address'] }}"
 
+##########################
+# keycloak
+
+keycloak_traefik: true
+
 ##########################
 # netbox
 
 netbox_traefik: true
 
-netbox_host: "{{ hostvars[inventory_hostname]['ansible_' + internal_interface]['ipv4']['address'] }}"
-
 ##########################
 # nexus
 
 nexus_traefik: true
 
-nexus_host: "{{ hostvars[inventory_hostname]['ansible_' + internal_interface]['ipv4']['address'] }}"
-
 netbox_plugins_extra:
```

Comment:
While you are cleaning up, move this into the netbox section?

Reply:
Same.